rust-src/concordium_base/src/id/types.rs

```diff
@@ -2100,6 +2102,34 @@ impl AccountKeys {
         }
         signatures
     }
+
+    /// Sign the provided data with all of the keys in [`AccountKeys`].
+    /// The thresholds are ignored. The return type is [`AccountSignatures`].
```

Comment:
I don't think it's necessary to say that the return type is that. It's clear from the type signature :)

Reply:
I added it explicitly because the `sign_data()` function has the same description without it. the last sentence gives a hint what the difference between `sign_data()` and `sign_message()` function is.